shell_test.go

```diff
@@ -398,7 +398,7 @@ func TestDagPutWithOpts(t *testing.T) {
 
 	c, err := s.DagPutWithOpts(`{"x": "abc","y":"def"}`, options.Dag.Pin("true"))
 	is.Nil(err)
-	is.Equal(c, "bafyreidrm3r2k6vlxqp2fk47sboeycf7apddib47w7cyagrajtpaxxl2pi")
+	is.Equal(c, "bafireidrm3r2k6vlxqp2fk47sboeycf7apddib47w7cyagrajtpaxxl2pi")
```

Comment:
Why are these CIDs different?

Reply:
I believe this is the codec, which we now account for correctly as `dag-json` rather than `json`

Reply:
https://cid.ipfs.io/#bafireidrm3r2k6vlxqp2fk47sboeycf7apddib47w7cyagrajtpaxxl2pi
https://cid.ipfs.io/#bafyreidrm3r2k6vlxqp2fk47sboeycf7apddib47w7cyagrajtpaxxl2pi

Are you sure, it looks like it's because you changed the output encoding from cbor to dag-cbor. This shouldn't be needed, right?

Reply:
sorry, i knew it was adding the `dag`.
The default is now `dag-cbor` for the encoding. (in that we don't fail if you try to put in a cid link in your dag put)

Reply:
IIUC what's happening here is that we used to say `cbor = dagcbor` and the default is `cbor`. We now have `cbor` as a separate option from `dag-cbor`, but the default has changed to `dag-cbor`.

I'd think that if you revert this change, but just temporarily bump CI to point the go-ipfs target branch that things will pass.

Reply:
note: reading this again, i now believe this shouldn't pass without this change: line 424 of the current master of this repo explicitly overrides the default back to `cbor`.